src/style.css

```diff
@@ -0,0 +1,103 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf) format("truetype");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf) format("truetype");
+  font-weight: 300;
+}
+
+.BigForm {
```

Comment:
it isn't a good practice to name classes with a big letter

Reply:
corrected